src/test/resources/Homework18.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="Homework18" verbose="1" >
+<test name="playSongTest" >
+    <classes>
+<!--        <class name="LoginTests"/>-->
+        <class name="BaseTest"/>
+        <class name="Homework18">
+            <methods>
+                <include name="playSongTest"/>
+            </methods>
+        </class>
+    </classes>
+</test>
+</suite>
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/226677967-707d6266-2198-4d27-8538-09d9475ad485.png)